swri_roscpp/include/swri_roscpp/dynamic_parameters.h

```diff
@@ -59,6 +59,7 @@ namespace swri
     Type type;
     std::string name;
     std::string description;
+    std::vector<std::pair<std::string, int>> enums;
```

Comment:
Not having a space between the `>`'s will break some versions of gcc, although I'm not sure if that's an issue on Ubuntu 16.04 / 18.04.
```suggestion
    std::vector<std::pair<std::string, int> > enums;
```